cas-server-documentation/installation/Pac4j-Authentication.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: default
```

Comment:
New documentation to explain how to use a pac4j authenticator wrapped in a CAS authentication handler.